### PR TITLE
Better explain dependency link processing

### PR DESCRIPTION
**Pull request by evandandrea**
As covered in https://github.com/CanonicalLtd/snappy-docs/pull/87#discussion_r126335736, better explain dependency link processing.

### CODE REVIEW

File: snapcraft/plugins/python.py [hunk 1]
```diff
@@ -36,7 +36,8 @@
       Path to a constraints file
     - process-dependency-links:
       (bool; default: false)
-      Enable the processing of dependency links.
+      Enable the processing of dependency links in pip, which allow one
```
Comment (sergiusens) [inline snapcraft/plugins/python.py]: allows?

File: snapcraft/plugins/python2.py [hunk 2]
```diff
@@ -39,7 +39,8 @@
       Path to a constraints file
     - process-dependency-links:
       (bool; default: false)
-      Enable the processing of dependency links.
+      Enable the processing of dependency links in pip, which allow one
```
Comment (sergiusens) [inline snapcraft/plugins/python2.py]: allows?

File: snapcraft/plugins/python3.py [hunk 3]
```diff
@@ -39,7 +39,8 @@
       Path to a constraints file
     - process-dependency-links:
       (bool; default: false)
-      Enable the processing of dependency links.
+      Enable the processing of dependency links in pip, which allow one
```
Comment (sergiusens) [inline snapcraft/plugins/python3.py]: allows?

File: snapcraft/plugins/python.py (hunk 1 above)
Comment (evandandrea) [inline snapcraft/plugins/python.py]: You wouldn't say "dependency links allows one project to provide …", you'd say "dependency links allow one project to provide…"

Comment (sergiusens) [inline snapcraft/plugins/python.py]: right, I read without the back context.

**Pull request merged**